specification/contosowidgetmanager-New/resource-manager/Microsoft.Contoso/Contoso.Management/employee.tsp

```diff
@@ -0,0 +1,63 @@
+import "@typespec/rest";
```

Comment:
@allenjzhang TSV failing currently because you have a capital letter in the folder name update to lower case `New`

Reply:
I put up another PR with the folder renamed as a test:
https://github.com/Azure/azure-rest-api-specs/pull/34537